odoo/addons/base/ir/ir_model.py

```diff
@@ -912,10 +912,13 @@ def _add_manual_fields(self, model):
         fields_data = self._get_manual_field_data(model._name)
         for name, field_data in fields_data.items():
             if name not in model._fields and field_data['state'] == 'manual':
-                field = self._instanciate(field_data)
-                if field:
-                    model._add_field(name, field)
-
+                try:
+                    field = self._instanciate(field_data)
+                    if field:
+                        model._add_field(name, field)
+                except:
+                    _logger.warning("Manual field '{}' : loading skipped".format(
```

Comment:
What about adding it to the _bad_fields ? Isn't it possible?

Reply:
Well, in one place it's the field object which is set in `_bad_fields`, in the other it's just the name:
```
type(self)._bad_fields[name] = name
```
against
```
cls._bad_fields[name] = cls._fields[name]  # returns the object field, not just the name
```
It can be one or the other, as we don't really use anything else for the moment than the name, but it can't be both.
In addition, if you choose to only store the name, than it can be a simple `set` rather than a dict, as storing
`{'x_studio_XZtu': 'x_studio_XZtu'}` is a pointless.